microsoft-edge/webview2/concepts/windowed-vs-visual-hosting.md

```diff
@@ -48,12 +48,6 @@ With visual hosting, for example you must choose how when the user resizes the w
 Key compatibility limitations include the operating system and rendering in framework and non-framework apps.
 
 
-<!-- ------------------------------ -->
-#### Operating systems
-
-Windows 7 and Windows 8 can only do windowed hosting, not visual hosting.
```

Comment:
General comment: Instead of saying Win7/8/8.1 are not supported we should mention that WebView2 Runtime version 109 is the last to support those operating systems and SDK version 1.0.1519.0 and higher will no longer support those operating systems. Similar to how it is mentioned here: https://learn.microsoft.com/en-us/deployedge/microsoft-edge-supported-operating-systems. @champnic what do you think?

Reply:
GitHub's rendered view, of section "Introduction to Microsoft Edge WebView2 > Supported platforms > Windows 7 and 8":
https://github.com/mikehoffms/edge-developer/blob/user/mikehoffms/win7win8/microsoft-edge/webview2/index.md#windows-7-and-8

Reply:
Restored this "Operating systems" section.  In this section, linked to topmost intro article's new Win7/8 heading.